Makefile

```diff
@@ -1,6 +1,6 @@
 DOCKER_REPO = phpmyadmin/phpmyadmin
 
-.PHONY: all build build_nc run logs clean stop rm prune
+.PHONY: all build build-apache build-fpm build-fpm-alpine run logs clean stop rm prune
```

Comment:
Just remove `build_nc`, everything else is already done in `build`.

Reply:
Thanks !

Reply:
Yeah, I know everything else is in `build`, but I have added them separately so that we can also run `make build-apache` for the cases where we just need to execute `build-apache` rather executing everything.

Reply:
Let me know this, if it isn't the requirement I will fix accordingly.

Reply:
:fast_forward: @williamdes

Reply:
@ankitjain28may Can you please apply the suggestion of @J0WI on this PR ?

Reply:
Sure